tests/suite/test_suite.py

```diff
@@ -126,6 +127,8 @@
     # test the most complete ledgers
     governance_history.test_ledger_is_readable,
     governance_history.test_tables_doc,
+    # e2e_operations:
+    e2e_operations.test_forced_ledger_chunk,
```

Comment:
Could you move this up as the previous `governance_history` tests should be run last.